packages/default/scss/button/_layout.scss

```diff
@@ -93,7 +93,7 @@
                 align-items: center;
                 justify-content: center;
 
-                &.k-svg-icon > svg,
+                > svg,
```

Comment:
we could remove the > svg selector here

Reply:
We need this selector, otherwise the icons increases its width/height